Stripe/ApplePay/STPAPIClient+ApplePay.m

```diff
@@ -68,6 +68,92 @@ + (NSData *)formEncodedDataForPayment:(PKPayment *)payment {
         }
     }
 
+    if (payment.shippingAddress && payment.billingAddress == nil) {
+        NSMutableDictionary *params = [NSMutableDictionary dictionary];
```

Comment:
There's a lot of repeated code here. It'd be better to extract this into a function:

```objc
- (NSString *)queryStringWithAddress:(ABRecord *)record {
...
}
```